services.html

```diff
@@ -0,0 +1,135 @@
+                sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.
+                <div class="more more3__Learn--More">
+                    Learn More </div>
+            </p>
+        </div>
+        <div class="box box4">
+            <h2>Web Development</h2>
+            <p> Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris
+                consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque,
+                sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.
+                <div class="more more4__Learn--More">
+                    Learn More </div>
+            </p>
+        </div>
+        <div class="box box5">
+            <h2> iOS Development</h2>
+            <p> Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris
+                consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque,
+                sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.
+                <div class="more more5__Learn--More">
+                    Learn More </div>
+            </p>
+        </div>
+        <div class="box box6">
+            <h2>Android Development </h2>
+            <p> Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris
+                consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque,
+                sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.
+                <div class="more more6__Learn--More">
+                    Learn More </div>
+            </p>
+        </div>
+    </div>
+
+
+    <div class="bottom">
+        <div class="fact fact1">
+            <h2>Some Facts About Our Services</h2>
+            <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris
+                consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque,
+                sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.
+            </p>
+            <ul>
+                <li>Awesome thing</li>
+                <li>Amazing thing</li>
+                <li>Cool thing</li>
+                <li>Great thing</li>
+            </ul>
+        </div>
+        <div class="fact fact3">
+            <img class="services-info-img" src="img/services-info.png" alt="one of our employees hard at work">
+        </div>
+        <div class="fact fact2">
+
+        </div>
+    </div>
+    <footer>
+        <h3> Copyright Great Idea! 2018</h3>
+    </footer>
+</div>
+</body>
+
+</html>
```

Comment:
Your project looks great.  The breakpoints and design are exactly as asked for.  Great job on this project Stefan.

Reply:
I am confused I am with PT2 class I hope you have the right student?